templates/index.html

```diff
@@ -11,6 +11,7 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
         <link rel="stylesheet" href="{% static 'css/index.css' %}">
+        <meta http-equiv="refresh" content="120" >
```

Comment:
You can remove this meta tag coz this one is not required

Reply:
Did the changes you asked for.
Thanks for pointing out the mistake